commerce-order-web/src/main/resources/META-INF/resources/order/emails.jsp

```diff
@@ -22,95 +22,133 @@ CommerceOrderEditDisplayContext commerceOrderEditDisplayContext = (CommerceOrder
 
 <commerce-ui:panel
 	bodyClasses="p-0"
-	title="emails"
+	title='<%= LanguageUtil.get(request, "emails") %>'
 >
-	<div class="list-group list-group-no-bordered mb-0 text-body">
-
-		<%
-		for (CommerceOrderNote commerceOrderNote : commerceOrderEditDisplayContext.getCommerceOrderNotes()) {
-		%>
+	<liferay-ui:search-container
+		id="commerceNotificationQueueEntries"
+		searchContainer="<%= commerceOrderEditDisplayContext.getCommerceNotificationQueueEntriesSearchContainer() %>"
+	>
+		<liferay-ui:search-container-row
+			className="com.liferay.commerce.notification.model.CommerceNotificationQueueEntry"
+			escapedModel="<%= true %>"
+			keyProperty="commerceNotificationQueueEntryId"
+			modelVar="commerceNotificationQueueEntry"
+		>
+
+			<%
+			User notificationUser = commerceOrderEditDisplayContext.getNotificationUser(commerceNotificationQueueEntry);
+
+			PortletURL rowURL = renderResponse.createRenderURL();
```

Comment:
Why have the `rowURL` logic up here, but the `data` logic down lower? Looks like they're used in the same place -- makes sense to me to declare them in the same scriptlet.